src/compas/topology/duality.py

```diff
@@ -12,6 +14,7 @@
     'mesh_dual',
     'network_dual',
     'network_find_faces',
+    'network_complement'
```

Comment:
i am reworking the structure of the packages and location of algorithms in a separate branch, so this might become an irrelevant question at some point, but just for argument's sake i will ask it anyway, is `duality.py` the right location for this?

Reply:
I put it here because of the relation with the original graph, which can be compared to the one between a primal and a dual graph. But it's not duality. So I added a new location for complementarity.

Reply:
cool but it seems you now need to resolve a few conflicts with the base branch :)